src/sidebar/components/test/SelectionTabs-test.js

```diff
@@ -49,54 +58,130 @@ describe('SelectionTabs', function () {
+
       assert.isTrue(tabs.at(0).hasClass('is-selected'));
       assert.equal(tabs.at(0).prop('aria-selected'), 'true');
       assert.equal(tabs.at(1).prop('aria-selected'), 'false');
     });
 
-    it('should display notes tab as selected', function () {
+    it('should not display the add-page-note button when the annotations tab is active', () => {
+      fakeSettings.enableExperimentalNewNoteButton = true;
+      const wrapper = createComponent();
+      assert.equal(wrapper.find('LabeledButton').length, 0);
+    });
+  });
+
+  describe('Notes tab', () => {
+    it('should display notes tab as selected when it is active', () => {
       fakeStore.selectedTab.returns('note');
-      const wrapper = createComponent({});
+      const wrapper = createComponent();
+
       const tabs = wrapper.find('button');
+
       assert.isTrue(tabs.at(1).hasClass('is-selected'));
       assert.equal(tabs.at(1).prop('aria-selected'), 'true');
       assert.equal(tabs.at(0).prop('aria-selected'), 'false');
     });
 
-    it('should display orphans tab as selected if there is 1 or more orphans', function () {
+    describe('Add Page Note button', () => {
+      it('should not display the add-page-note button if the associated setting is not enabled', () => {
+        fakeSettings.enableExperimentalNewNoteButton = false;
+        fakeStore.selectedTab.returns('note');
+
+        const wrapper = createComponent();
+
+        assert.equal(wrapper.find('LabeledButton').length, 0);
+      });
+
+      it('should display the add-page-note button when the associated setting is enabled', () => {
+        fakeSettings.enableExperimentalNewNoteButton = true;
+        fakeStore.selectedTab.returns('note');
+
+        const wrapper = createComponent();
+
+        assert.equal(wrapper.find('LabeledButton').length, 1);
+      });
+
+      it('should apply background-color styling from settings', () => {
+        fakeSettings = {
+          branding: {
+            ctaBackgroundColor: '#00f',
+          },
+          enableExperimentalNewNoteButton: true,
+        };
+        fakeStore.selectedTab.returns('note');
+
+        const wrapper = createComponent();
+
+        const button = wrapper.find('LabeledButton');
+        assert.deepEqual(button.prop('style'), { backgroundColor: '#00f' });
+      });
+
+      it('should add a new page note on click', () => {
+        fakeSettings.enableExperimentalNewNoteButton = true;
+        fakeStore.selectedTab.returns('note');
+
+        const wrapper = createComponent();
+        wrapper.find('LabeledButton').props().onClick();
+
+        assert.calledOnce(fakeAnnotationsService.createPageNote);
+      });
+    });
+  });
+
+  describe('orphans tab', () => {
+    it('should display orphans tab if there is 1 or more orphans', () => {
+      fakeStore.orphanCount.returns(1);
+
+      const wrapper = createComponent();
+
+      const tabs = wrapper.find('button');
+      assert.equal(tabs.length, 3);
```

Comment:
Possible improvement, unrelated to this PR:
```suggestion
      assert.isTrue(wrapper.find("Tab[label='Orphans']").exists());
```

Reply:
I quite prefer this pattern. Tell you what: I have additional work to do in `SelectionTabs` (this component) during this sprint: there are a bunch of `button` elements that need to be converted to use components—I can make these tests better then!